src/main/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersController.java

```diff
@@ -1,7 +1,53 @@
 package uk.gov.hmcts.reform.cpo.controllers;
 
+import io.swagger.annotations.ApiParam;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.cpo.domain.CasePaymentOrder;
+import uk.gov.hmcts.reform.cpo.repository.CasePaymentOrderQueryFilter;
+import uk.gov.hmcts.reform.cpo.service.impl.CasePaymentOrdersServiceImpl;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
 
 @RestController
+@RequestMapping(value = "/api")
 public class CasePaymentOrdersController {
```

Comment:
Should we add "/api" ?  if not I can removed it

Reply:
the agreed endpoint on the scope document is:
`case-payment-orders/?ids=uuid1`
so I think we should remove it

Reply:
DONE